src/components/Trade/TradeConfirm/TradeConfirm.tsx

```diff
@@ -133,6 +133,12 @@ export const TradeConfirm = ({ history }: RouterProps) => {
     history.push('/trade/input')
   }
 
+  const tradeFiatAmount = toFiat(
+    bnOrZero(fromBaseUnit(bnOrZero(trade?.sellAmount), trade?.sellAsset.precision ?? 0))
```

Comment:
`bnOrZero()` will always do an `unwrap_or_default` style defaulting to 0 for falsy values, no need to use the nullish coercing operator to have a 0 expression on the falsy path